src/components/EditorWrapper.vue

```diff
@@ -182,7 +182,7 @@ export default {
 			return this.$store.state.showAuthorAnnotations
 		},
 		lastSavedStatus() {
-			return this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
+			return this.hasConnectionIssue ? t('text', 'Offline, changes will be saved when online') : this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
```

Comment:
Great improvement! Maybe the wording could be tweaked a bit:

```suggestion
			return this.hasConnectionIssue ? t('text', 'Connection lost. Will be saved once connection is restored.') : this.dirtyStateIndicator ? t('text', 'Saving …') : t('text', 'Saved')
```

Reply:
@jancborchardt What do you think about the message?

Reply:
Hmm, I think offline and online are a bit less technical and more commonly understood terms, so I'd keep them?

Reply:
Ok, no hard feelings here :blush: Funny sidefact: I thought that 'Connection lost' would be less technical and more accurate at the same time, that's why I suggested it :grimacing: